trpcultivate_phenotypes/src/Service/TripalCultivatePhenotypesGenusProjectService.php

```diff
@@ -150,17 +147,18 @@ public function getGenusOfProject($project) {
 
       // Fetch genus paired to a project. If multiple genus have been set prior,
       // restrict search to genus that are active/configured using this module.
-      $result = $this->chado_connection->query("
-        SELECT organism_id AS id, genus FROM {1:organism}
-        WHERE genus = (SELECT value::VARCHAR FROM {1:projectprop}
-          WHERE project_id = :project_id AND type_id = :type_id AND LOWER(value) IN (:active_genus[]) LIMIT 1)
-        LIMIT 1
-      ", [':project_id' => $project, ':type_id' => $this->sysvar_genus, ':active_genus[]' => $active_genus]);
-
-      $genus_project = $result->fetchObject();
+      $result = $this->chado_connection->select('1:projectprop', 'pp')
+        ->fields('pp', ['value'])
+        ->condition('pp.project_id', $project, '=')
+        ->condition('pp.type_id', $this->sysvar_genus, '=')
+        ->where('LOWER(pp.value) IN (:active_genus[])', [':active_genus[]' => $active_genus])
+        ->orderBy('value', 'ASC')
```

Comment:
Would it be more fitting here to also order by rank, to be consistent with the setter?